osu.Game.Rulesets.Mania/Skinning/Legacy/LegacyStageBackground.cs

```diff
@@ -126,8 +127,8 @@ private void load(ISkinSource skin)
                             },
                             new Container
                             {
+                                X = isLastColumn ? -0.16f : 0,
```

Comment:
This part also comes from stable (x1.6), wouldn't hurt to include it. See [stable reference](https://github.com/peppy/osu-stable-reference/blob/9a0748563812b5085a0ef5f8600b997408330eab/osu!/GameModes/Play/Rulesets/Mania/Stage/ColumnMania.cs#L157).